### PR TITLE
[MO] - [Parallel namespace test] -> RollingUpdateST

### PR DESCRIPTION
**Pull request by see-quick**
### Type of change

- Enhancement / new feature

### Description

This PR adding support for `RollingUpdatesT` suite to run in a parallel namespace mode.

### Checklist

- [x] Write tests
- [x] Make sure all tests pass

### CODE REVIEW

**Comment by see-quick** [comment]
After this PR https://github.com/strimzi/strimzi-kafka-operator/pull/4764 is merged rebase it.

File: systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaConnectResource.java [hunk 1]
```diff
@@ -33,7 +33,7 @@ public void create(KafkaConnect resource) {
         kafkaConnectClient().inNamespace(resource.getMetadata().getNamespace()).withName(resource.getMetadata().getName()).createOrReplace(resource);
     }
     @Override
-    public void delete(KafkaConnect resource) {
+    public void delete(KafkaConnect resource)  {
```
Comment (Frawless) [inline systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaConnectResource.java:36]: Revert this please

**Review by Frawless — APPROVED** [review]

**Review by scholzj — APPROVED** [review]

**Pull request merged**